### PR TITLE
Shuffle the setup directory a bit.

### PR DESCRIPTION
**Pull request by geeknoid**
I'm expecting some broken links I'll need to go fix, but you can see the general layout in the meantime.

### CODE REVIEW

**Comment by frankbu** [comment]
This looks really good to me. Additionally, I would suggest moving `Additional Setup Guides` under `Install`, since they are all install related.

File: content/docs/setup/kubernetes/additional-setup/cni/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ networking functionality but without requiring Istio users to enable elevated
 Kubernetes RBAC permissions.
 
 The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle's network
-setup phase, thereby removing the [`NET_ADMIN` capability requirement](/docs/setup/kubernetes/requirements//)
+setup phase, thereby removing the [`NET_ADMIN` capability requirement](/docs/setup/kubernetes/prepare/requirements//)
```
Comment (frankbu) [inline content/docs/setup/kubernetes/additional-setup/cni/index.md]:
```suggestion
setup phase, thereby removing the [`NET_ADMIN` capability requirement](/docs/setup/kubernetes/prepare/requirements/)
```

**Comment by rcaballeromx** [comment]
> This looks really good to me.

Agreed

> Additionally, I would suggest moving `Additional Setup Guides` under `Install`, since they are all install related.

IMO they are setup related but not necessarily install related. Readers might need to expand the mesh or manually inject sidecars long after they completed the installation, for example.
I would leave them as they are in this PR.

**Comment by frankbu** [comment]
> IMO they are setup related but not necessarily install related. Readers might need to expand the mesh or manually inject sidecars long after they completed the installation, for example.
I would leave them as they are in this PR.

If we leave it, at least change the title to something short and sweet, like the rest of the sections, maybe:

* Prepare
* Download
* Install
* Upgrade
* More Guides

**Comment by geeknoid** [comment]
OK, all bad links fixed.

Should I check in?

**Pull request merged**